tests/test_check_stac_metadata.py

```diff
@@ -581,10 +581,48 @@ def should_raise_exception_if_non_s3_url_is_passed() -> None:
         STACDatasetValidator(url_reader, MockValidationResultFactory()).run(https_url)
 
 
-def should_return_assets_from_validated_metadata_files(subtests: SubTests) -> None:
+def should_return_assets_from_validated_collection_metadata_files(subtests: SubTests) -> None:
```

Comment:
I don't think "return" in the name is appropriate - we're asserting on a field of the `validator`. Ditto in the next test, I think.